app/role_data.go

```diff
@@ -291,6 +294,10 @@ func addRoles(management *config.ManagementContext) (string, error) {
 	rb.addRoleTemplate("View Project Catalogs", "projectcatalogs-view", "project", true, false, false, false).
 		addRule().apiGroups("management.cattle.io").resources("projectcatalogs").verbs("get", "list", "watch")
 
+	rb.addRoleTemplate("Project Monitoring View Role", "project-monitoring-readonly", "project", true, false, true, false).
```

Comment:
Nit: To follow the convention of other roleTemplates, we can name it as `View Project Monitoring`, `projectmonitoring-view`.

Reply:
`View Project Monitoring` does not explain the role template here so I name it as the owner's way.

Reply:
i agree with @gitlawr. The break from convention doesn't fit well.